js/src/forum/components/LogInModal.js

```diff
@@ -61,11 +61,12 @@ export default class LogInModal extends Modal {
 
   body() {
     return [
-      <LogInButtons/>,
+      <LogInButtons />,
 
-      <div className="Form Form--centered">
-        {this.fields().toArray()}
-      </div>
+      app.forum.attribute('enablePasswordAuth') ?
```

Comment:
Wouldn't it be better to add the condition inside of `fields()` ? And return an empty fields array when password login is disabled.

That way extensions that want to insert custom fields into the login modal by `extend()`ing the `fields` ItemList can still do it even if password auth is disabled.

Reply:
Done, made the same fix for signup modal